ark/utils/spatial_analysis_utils_test.py

```diff
@@ -143,15 +139,13 @@ def test_compute_close_cell_num():
     # Only include the columns of markers
     fov_channel_data = all_data.drop(all_data.columns[[
         0, 1, 14, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32]], axis=1)
-    # List of all markers
-    channel_titles = fov_channel_data.columns
-    # Length of marker list
-    channel_num = len(channel_titles)
+
     # Subsetting threshold matrix to only include column with threshold values
     thresh_vec = example_thresholds.iloc[0:20, 1]
 
-    example_closenum, m1 = spatial_analysis_utils.compute_close_cell_num(
-        dist_mat=example_dist_mat, dist_lim=100, num=channel_num, analysis_type="channel",
+    # TODO: shouldn't we have a test somewhere for cluster analysis type?
```

Comment:
Yes. I think we already have example data that should work for this in the test function for calc_cluster_spatial_enrichment. If not, even just a simple test that runs the function with the right inputs and doesn't check for correctness would be good to add.